tests/cupyx_tests/scipy_tests/interpolate_tests/test_polyint.py

```diff
@@ -0,0 +1,210 @@
+import numpy
+import pytest
+
+import cupy
+from cupy import testing
+import cupyx.scipy.interpolate  # NOQA
+
+try:
+    from scipy import interpolate  # NOQA
+except ImportError:
+    pass
+
+
+@testing.with_requires("scipy")
+class TestKrogh:
+
+    @testing.for_all_dtypes(no_bool=True, no_complex=True)
+    @testing.numpy_cupy_allclose(scipy_name='scp', atol=1e-7, rtol=1e-7)
+    def test_lagrange(self, xp, scp, dtype):
+        if xp.dtype(dtype).kind in 'iu':
```

Comment:
All signed and unsigned integer dtypes need to be skipped?

Reply:
Yes, we need this.

Reply:
Signed integers pass the test on my local. Would you also give it a try?

Reply:
Oo yes, you are right. Will update this.